guides/introduction/installation.md

```diff
@@ -138,6 +138,8 @@ latest javascript, then force an install.
 (cd assets && npm install --force phoenix_live_view)
 ```
 
+Make sure you have the `<%= csrf_meta_tag() %>` in this file `/lib/my_app_web/templates/layout/app.html.eex` before the `</head>`
```

Comment:
Recommended change 👇 
---

Ensure you have placed a CSRF meta tag inside the `<head>` tag in your layout like so.

```html
<%= csrf_meta_tag() %>
```